compose.yml

```diff
@@ -68,6 +68,7 @@ services:
       dockerfile: ./Dockerfile
     environment:
       BACKEND_URL: ${BACKEND_URL}
+      WS_URL: ${WS_URL}
```

Comment:
Should this `WS_URL` property also be added to the `.env.example` template file?